scripts/nginx/jellyfin.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+# Nginx Configuration for Jellyfin
+# Author: liara
+# Copyright (C) 2019 Swizzin
+# Licensed under GNU General Public License v3.0 GPL-3 (in short)
+#
+#   You may copy, distribute and modify the software as long as you track
+#   changes/dates in source files. Any modifications to our software
+#   including (via compiler) GPL-licensed code must also be made available
+#   under the GPL along with build & install instructions.
+# username="$(cat /root/.master.info | cut -d: -f1)"
+username="$(cut </root/.master.info -d: -f1)"
+ip_address="$(curl -s4 icanhazip.com)"
+#
+if [[ -n "$1" && ! -f /install/.jellyfin.lock ]]; then
+  app_port_https="$2"
+  sed -r 's#<WanDdns>(.*)</WanDdns>#<WanDdns>https://'"${ip_address}"'/jellyfin</WanDdns>#g' -i "/home/${username}/.config/Jellyfin/config/system.xml"
+  sed -r 's#<string>0.0.0.0</string>#<string>127.0.0.1</string>#g' -i "/home/${username}/.config/Jellyfin/config/system.xml"
+fi
+#
+if [[ -z "$1" && -f /install/.jellyfin.lock ]]; then
+  service jellyfin stop
+  app_port_https="$(sed -rn 's#(.*)<HttpsPortNumber>(.*)</HttpsPortNumber>#\2#p' "/home/${username}/.config/Jellyfin/config/system.xml")"
+  sed -r 's#<WanDdns>(.*)</WanDdns>#<WanDdns>https://'"${ip_address}"'/jellyfin</WanDdns>#g' -i "/home/${username}/.config/Jellyfin/config/system.xml"
+  sed -r 's#<string>0.0.0.0</string>#<string>127.0.0.1</string>#g' -i "/home/${username}/.config/Jellyfin/config/system.xml"
+  service jellyfin start
+fi
```

Comment:
I don't see the need for an `-n` and a `-z` let's just assume `-z` always because I currently see no use case for calling the nginx script with an argument when the standard for all other nginx scripts is without.